test/build/50_dockerfile.bats

```diff
@@ -471,6 +479,10 @@ EOF
   echo "$output"
   [[ $status -eq 0 ]]
   diff -u <(echo "$env_expected") <(echo "$output" | grep -E "^\('chse_")
+
+  if [[ $CH_BUILDER = ch-image ]]; then
+    ch-image delete env-syntax
+  fi
```

Comment:
So this raises something interesting I had not thought of — it seems awkward to delete the images we don't need for only `ch-image` (and leave them for other builders). I think there are at least two options:

1. Write several lines of code for every multi-builder test to select the right delete command. You'd probably want to include Buildah but it won't be tested.
2. Change all these tests to use the *same* temporary image name — and don't delete it. Also document the name in the contributor's guide. That will give us one image hanging around, which seems OK.

What do you think?

Reply:
I am leaning towards the 2nd idea. It seems cleaner and nicer to use one variable name that overwrites the image vs adding extra code to decide how to delete said image.

Reply:
![make it so](https://media.giphy.com/media/VLoN2iW8ii3wA/giphy.gif)